### DISCUSSION TITLE
posts/Go/gen/

### DISCUSSION THREAD

**Discussion by giscus** [Q1mi/BlogComments#373] [Announcements]
# posts/Go/gen/

李文周的 Blog 提供 Go 语言中文教程,李文周博客,Go 语言学习之路是我整理的一个免费教程，包含部分视频教程，也是非常适合新手学习Go语言的入门教程。本文介绍了如何使用基于GORM的安全ORM框架——Gen的基本使用，本文算是一篇Gen框架使用指南。

https://www.liwenzhou.com/posts/Go/gen/

**Comment by fangao888** [comment]
写的真好，感谢李老师
